sec4-notebook.tex

```diff
@@ -1,8 +1,57 @@
 \section{The Notebook Aspect}\label{notebook-aspect}
 
+The Science Platform offers in-house scientists and engineers as well as (in operations) general science users the opportunity to interactively explore the LSST dataset from a python notebook environment.
+
 \subsection{JupyterHub / JupyterLab service}\label{jupyterhub-jupyterlab-service}
 
+The notebook service is a scalable kubernetes deployment with JupyterHub spawning individual JupyterLab individual pods for each user. The basic architecture is outlined in \figref{fig:jlarch}. JupyterLab is a rich notebook architecture that offers terminal access and dashboarding in addition to the traditional Jupyter Notebook concept, and
+
+A user of the JupyterLab sevice will hence be able to interact with LSST services in the same way they might have with python scripts in a shell environment. They can use our python interfaces to the DAX and the Butler for data access, the interface to the workflow system for batch processing, visualisation via bokeh, Firefly, matplotlib etc. 
+
+The capabilities of the service are tied to major project milestones outlined in the table below.
+
+\begin{footnotesize}
+% \begin{longtable}[c]{|l|l|p{0.4\textwidth}|l|}
+\begin{longtable}[c]{|l|l|l|l|}
+\toprule
+Version & Audience & Target L2 Milestone & Shims ok?\tabularnewline
+\midrule
+\endhead
+proto & SQuaRE and Arch & none & y\tabularnewline
+alpha & DM devs, Commissioning team training & LDM-503-1 &
+y\tabularnewline
+beta & Commissioning Team & LDM-503-9 & y\tabularnewline
+1.0 & Commissioning (ComCam) Capabilities & LDM-503-12 &
+n\tabularnewline
+2.0 & Science Verification Capabilities & LDM-503-14 & n\tabularnewline
+3.0 & General Science User Capabilities & LDM-503-16 & n\tabularnewline
+\bottomrule
+\end{longtable}
+\end{footnotesize}
+
+Note in particular that unlike the originally planned ``Level 3'' science user capabilities, the JL service is the primary interface of the LSST System Scientist to Commissioning. This introduces scope not required by the general science user, eg. timely access to data in the Engineering Facilities Database and the ability to rapidly deploy instances of the service on the Commissioning Cluster.
```

Comment:
I'm not sure about the "unlike" here.  I think "in addition to" would be better.